src/android/CallNotification.java

```diff
@@ -116,6 +118,9 @@ public void show() {
         this.timeoutRunnable = new Runnable() {
             @Override
             public void run() {
+                Log.d(TAG, "call missed, closing connection and call notification,");
+                Connection conn = MyConnectionService.getConnectionByPayload(pushMessagePayload);
+                conn.setDisconnected(new DisconnectCause(DisconnectCause.MISSED));
```

Comment:
So I believe in theory this is not needed, as if the callee neither answers or declines the call, the call I think is eventually dismissed, and we should eventually receive an FCM data messsage with payload.dismiss = true.

If that doesn't happen for any reason though I worry that this would leave the connection in existance in a ringing state.

This will eventually cause new incoming connections to fail to be added, due to the MAX_RINGING_CALLS android telecom limit.

Have a feeling this is still a good idea for robustness at a minimum, to avoid relying on the backend telling us the call is dimissed